Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -405,9 +407,19 @@ define([
         this._pickPrimitive = createCommand(function() {
             that.pickPrimitiveActive = !that.pickPrimitiveActive;
             if (that.pickPrimitiveActive) {
-                that._canvas.addEventListener('mousedown', pickPrimitive, false);
+                if (FeatureDetection.supportsPointerEvents()) {
+                    that._canvas.addEventListener('pointerdown', pickPrimitive, true);
+                } else {
+                    that._canvas.addEventListener('mousedown', pickPrimitive, true);
+                    that._canvas.addEventListener('touchstart', pickPrimitive, true);
```

Comment:
`touchstart` doesn't return `e.clientX`, it has a list of touches with a clientX per touch.  I think each API needs to be handled separately here, unless you switch to Cesium's built-in event handling.